octoprint_TimeToFilament/__init__.py

```diff
@@ -66,6 +79,33 @@ def on_startup(self, host, port):
   @octoprint.plugin.BlueprintPlugin.route("/get_settings_defaults", methods=["GET"])
   def get_settings_defaults_as_string(self):
     return json.dumps(self.get_settings_defaults())
+    
+  def is_blueprint_csrf_protected(self):
+    return True
+    
+  @octoprint.plugin.BlueprintPlugin.route("/", methods=["GET"])
```

Comment:
If we use the `/` URL then it will make future URLs awkward.  Maybe it's better to name this URL.  And then if we want future endpoints, those can have different names.  You could call it `/get_cache` like the function or just `/get` or something else...

What do you think?

Reply:
How about `/api`? I would rename the function then, too.